src/current/_includes/v23.2/ui/insights.md

```diff
@@ -167,17 +186,23 @@ The Workload Insights tab surfaces the following type of insights:
 The transaction or statement execution failed.
 
 {% if page.cloud != true -%}
-The following screenshot shows a failed statement execution:
+The following screenshot shows a failed transaction execution:
+
+<img src="{{ 'images/v23.2/transaction_executions_failed.png' | relative_url }}" alt="Failed transaction execution" style="border:1px solid #eee;max-width:100%" />
 
-<img src="{{ 'images/v23.2/statement_executions_failed.png' | relative_url }}" alt="Failed statement execution" style="border:1px solid #eee;max-width:100%" />
+The following screenshot shows the default details of the preceding failed transaction execution.
 
-The following screenshot shows the execution details of the preceding failed statement execution.
+<img src="{{ 'images/v23.2/transaction_execution_details_failed.png' | relative_url }}" alt="Failed transaction execution details" style="border:1px solid #eee;max-width:100%" />
 
-<img src="{{ 'images/v23.2/statement_execution_details_failed.png' | relative_url }}" alt="Failed statement execution details" style="border:1px solid #eee;max-width:100%" />
+The following screenshot shows the conditional details of the preceding failed transaction execution.
+
+<img src="{{ 'images/v23.2/transaction_execution_details_failed_conditional.png' | relative_url }}" alt="Failed transaction execution details conditional" style="border:1px solid #eee;max-width:100%" />
 
 {% endif -%}
 
-The **Insights** column shows the name of the insight, in this case **Failed Execution**. The **Details** column provides the **Error Code**, such as `40001`. CockroachDB uses [PostgreSQL Error Codes](https://www.postgresql.org/docs/current/errcodes-appendix.html). `40001` indicates a `serialization_failure`.
+The **Insights** column shows the name of the insight, in this case **Failed Execution**. The **Details** column provides the **Error Code** and **Error Message**. CockroachDB uses [PostgreSQL Error Codes](https://www.postgresql.org/docs/current/errcodes-appendix.html).
+
+To capture more information in the event of a failed transaction execution due to a serialization conflict, set the `sql.contention.record_serialization_conflicts.enabled` [cluster setting]({{ link_prefix }}cluster-settings.html) to `true`. With this setting enabled, when the **Error Code** is `40001` and the **Error Message** specifically has [`RETRY_SERIALIZABLE`]({{ link_prefix }}transaction-retry-error-reference.html#retry_serializable)` - failed preemptive refresh due to conflicting locks`, a conditional **Failed Execution** section is displayed with **Conflicting Transaction** and **Conflicting Location** information.
```

Comment:
"and the Error Message specifically has RETRY_SERIALIZABLE- failed preemptive refresh due to conflicting locks"

Do we need to say " failed preemptive refresh due to conflicting locks"? This makes it seem like there are other forms of 40001 - **RETRY_SERIALIZABLE** errors. If there are multiple RETRY_SERIALIZABLE errors we should update this document to be more specific (and prescriptive) about the other RETRY_SERIALIZABLE error(s):

https://www.cockroachlabs.com/docs/stable/transaction-retry-error-reference#retry_serializable

Reply:
I updated the [RETRY_SERIALIZABLE section of the Transaction Retry Error Reference page](https://deploy-preview-17992--cockroachdb-docs.netlify.app/docs/v23.2/transaction-retry-error-reference.html#retry_serializable) based on @aadityasondhi and @arulajmani 's [suggestions](https://cockroachlabs.atlassian.net/browse/DOC-8027?focusedCommentId=195737).